test/compflags/bradc/help/userhelp.good

```diff
@@ -1,5 +1,27 @@
 Usage: chpl [flags] [source files]
 
+Primary Configuration Options:
+      --home <path>                   Path to Chapel's home directory
```

Comment:
Didn't notice this before, but do we want these at the top of `--help`? Not really a question that has to be answered for this PR since it's easy enough to change later, but to me these seem like they should be at the end